backend/FwHeadless/Services/SendReceiveHelpers.cs

```diff
@@ -100,7 +100,7 @@ public static async Task<LfMergeBridgeResult> SendReceive(FwDataProject project,
             { "languageDepotRepoName", "LexBox" },
             { "languageDepotRepoUri", repoUrl.AbsoluteUri },
             { "deleteRepoIfNoSuchBranch", "false" },
-            { "user", "LexBox" },
+            { "user", "Fieldworks Lite" }, // Not necessary if username was set at clone time, but why not
```

Comment:
```suggestion
            { "user", "FieldWorks Lite" }, // Not necessary if username was set at clone time, but why not
```

Reply:
Just caught that myself before seeing your comment. Thanks for being on the ball.